REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.4
-Compat
+julia 0.5
+Compat 0.18
```

Comment:
Thanks Tony.  I am constantly amazed at the number of tasks, like checking this pull request, that you can accomplish simultaneously